frontend/src/app/components/about/about.component.html

```diff
@@ -194,7 +68,7 @@ <h3 i18n="about.maintainers">Project Maintainers</h3>
       The Mempool Open Source Project
     </div>
     <p>
-      <a href="https://github.com/mempool/mempool">The Mempool Open Source Project</a> is free software; you can redistribute it and/or modify it under the terms of (at your option) either:<br>
+      <a href="https://github.com/Rucknium/bchmempool">The Mempool Open Source Project</a> is free software; you can redistribute it and/or modify it under the terms of (at your option) either:<br>
```

Comment:
What to do with copyrights and licenses?

Reply:
I think the structure of the licenses can be kept the same, i.e. keep the AGPL and GPL licenses.

I think "Copyright © 2019-2021
The Mempool Open Source Project "
should be changed to 
"Copyright © 2019-2022
The Mempool Open Source Project and the BCH Mempool Open Source Project"

The Trademark Notice can be eliminated since we aren't trademarking anything. Once the new logos come in, we will be brought into compliance with the BTC Mempool project's trademark policy.

Thank you.